vendor/github.com/twmb/franz-go/pkg/kadm/errors.go

```diff
@@ -121,3 +121,14 @@ func (e *ShardErrors) Error() string {
 	}
 	return fmt.Sprintf("request %s has %d separate shard errors, first: %s", e.Name, len(e.Errs), e.Errs[0].Err)
 }
+
+// Unwrap returns the underlying errors.
+func (e *ShardErrors) Unwrap() []error {
```

Comment:
TIL The [`errors` package][1] supports a variant of the unwraper interface, which returns a slice of errors from its `Unwrap`. I've never noticed that in the docs.

[1]: https://pkg.go.dev/errors#pkg-overview

Reply:
It's a recent feature (don't remember which golang version introduced it tho).